projects/ardin_mb/mb_memory.cc

```diff
@@ -112,7 +112,7 @@ void MBMemory::train(const cv::Mat &image)
     present(image, true);
 }
 //----------------------------------------------------------------------------
-float MBMemory::test(const cv::Mat &image) const
+float MBMemory::test(const cv::Mat &image)
```

Comment:
So, previously, this had to be const because it was inheriting from ``BoBRobotics::Navigation::VisualNavigationBase`` which was annoying because it required a vast amount of mutable state to store spikes etc. Now, because the project-specific ``VisualNavigationBase`` class allows algorithms to be stateful we can just make it non-const....does that make sense?

Reply:
Yep! Having ``mutable``s all over the place gets a bit cumbersome after a while...